noetic/distribution.yaml

```diff
@@ -1091,14 +1091,30 @@ repositories:
     doc:
       type: git
       url: https://github.com/4am-robotics/cob_common.git
-      version: kinetic_release_candidate
+      version: noetic-release-candidate
     release:
       packages:
       - cob_actions
       - cob_common
-      - cob_description
       - cob_msgs
       - cob_srvs
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/4am-robotics/cob_common-release.git
+      version: 0.7.11-1
+    source:
+      type: git
+      url: https://github.com/4am-robotics/cob_common.git
+      version: noetic-devel
+    status: maintained
+  cob_common_legacy:
```

Comment:
I hope I'm doing the right thing here - could you please confirm that this PR will:
- release a new version `0.7.11` of `cob_actions`, `cob_common`, `cob_msgs` and `cob_srvs` from `noetic-release-candidate`
- keep the last release `0.7.10` for `cob_description` and `raw_description` available based on `kinetic_release_candidate`

Reply:
Thank you for the feedback - I guess the other linked PRs are then also what I intend.
Thus, go ahead with the review and merge process.